app/views/content/blog/transferring-my-skills-to-teaching.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Transferring my skills to teaching
+date: "2020-02-01"
+author: Will Fordham
+images:
+  will-fordham:
+    path: "media/images/content/blog/stories-will.jpg"
+    thumbnail_path: "media/images/content/blog/stories-will.jpg"
+description: |-
+  Teaching trainee Will Fordham was working for a charity as a communications manager, when he realised that teaching was what he loved.
```

Comment:
```suggestion
  Teaching trainee Will Fordham was working as a communications manager, when he realised that teaching was what he loved.
```
Charity is only mentioned in a quote on the original my story - quote hasn't been included in the blog post - suggest we remove from meta description @EllieNodder ?